jobs/parser-config-lfc/templates/logstash-filters-custom.conf.erb

```diff
@@ -0,0 +1 @@
+<%= p('logstash_parser.custom_filters').empty? ? ' ' :  p('logstash_parser.custom_filters')  %>
```

Comment:
Is the space rather than an empty string intensional here? ... it seems harmless given the context but looked unusual

Reply:
It was part of Colin's changes that have been forward ported.

Reply:
Ignore this. It's fine :)